giskard/models/talk_result.py

```diff
@@ -0,0 +1,36 @@
+from dataclasses import dataclass
+
+from giskard.llm.client import LLMMessage
+
+
+@dataclass
+class TalkResult:
+    """The dataclass containing the result of the 'talk' call.
+
+    Attributes
+    ----------
+    response : LLMMessage
+        The response to the user's query.
+    summary : LLMMessage
+        The summary of the conversation between the user and the LLM agent.
+    """
+
+    response: LLMMessage
+    summary: LLMMessage
+
+    def __str__(self) -> str:
```

Comment:
I never remember, but is it on purpose not to change `__repr__`

Reply:
I will duplicate the __str__ body and create __repr__ method.

Reply:
I replaced `__str__` by `__repr__`